pkg/auth/default.go

```diff
@@ -32,18 +31,18 @@ func NewDefaultCredentials() (credentials.PerRPCCredentials, error) {
 }
 
 // StoreCredentialsFile stores the API key at the given path to a well known location.
-func StoreCredentialsFile(path string) {
-	content, err := ioutil.ReadFile(path)
+func StoreCredentialsFile(path string) error {
+	content, err := os.ReadFile(path)
 	if err != nil {
-		log.Fatalf("could not read credentials file.\n%v", err)
+		return fmt.Errorf("could not read credentials file: %w", err)
 	}
 
-	config.EnsureConfigDir()
+	_ = config.EnsureConfigDir()
```

Comment:
WriteFile would fail if this produced an error (no config dir). Perhaps we should return this err instead of ignoring it?

Reply:
I believe the write file would blow up with enough details on why (the directory it tried to write to and why it couldn't). And there's fatals that might get called within `EnsureConfigDir` that will have other details. I'm mixed on if this would provide useful feedback which is why I didn't change this. (it was ignoring the error previously, it will continue to ignore it now)